packages/blade/src/utils/makeAccessible/accessibilityMapNative.ts

```diff
@@ -0,0 +1,99 @@
+import type { AccessibilityMap } from './makeAccessible.d';
+
+export const accessibilityValue = {
+  valueMax: 'max',
+  valueMin: 'min',
+  valueNow: 'now',
+  valueText: 'text',
+};
+
+export const accessibilityState = {
+  selected: 'selected',
+  disabled: 'disabled',
+  expanded: 'expanded',
+  busy: 'busy',
+  checked: 'checked',
+};
+
+export const accessibilityValueKeys = Object.keys(accessibilityValue);
+export const accessibilityStateKeys = Object.keys(accessibilityState);
+
+// TODO:
+// importantForAccessibility
+// accessibilityElementsHidden
+// accessibilityViewIsModal
+export const accessibilityMap: AccessibilityMap = {
+  ...accessibilityState,
+  ...accessibilityValue,
+  activeDescendant: 'accessibilityActiveDescendant',
+  atomic: 'accessibilityAtomic',
+  autoComplete: 'accessibilityAutoComplete',
+  colCount: 'accessibilityColCount',
+  colIndex: 'accessibilityColIndex',
+  colSpan: 'accessibilityColSpan',
+  controls: 'accessibilityControls',
+  describedBy: 'accessibilityDescribedBy',
+  details: 'accessibilityDetails',
+  errorMessage: 'accessibilityErrorMessage',
+  flowTo: 'accessibilityFlowTo',
+  hasPopup: 'accessibilityHasPopup',
+  hidden: 'accessibilityHidden',
+  invalid: 'accessibilityInvalid',
+  keyShortcuts: 'accessibilityKeyShortcuts',
+  label: 'accessibilityLabel',
+  labelledBy: 'accessibilityLabelledBy',
+  liveRegion: 'accessibilityLiveRegion',
+  modal: 'accessibilityModal',
+  multiline: 'accessibilityMultiline',
+  multiSelectable: 'accessibilityMultiSelectable',
+  orientation: 'accessibilityOrientation',
+  owns: 'accessibilityOwns',
+  placeholder: 'accessibilityPlaceholder',
+  posInSet: 'accessibilityPosInSet',
+  pressed: 'accessibilityPressed',
+  readOnly: 'accessibilityReadOnly',
+  required: 'accessibilityRequired',
+  role: 'accessibilityRole',
+  roleDescription: 'accessibilityRoleDescription',
+  rowCount: 'accessibilityRowCount',
+  rowIndex: 'accessibilityRowIndex',
+  rowSpan: 'accessibilityRowSpan',
+  setSize: 'accessibilitySetSize',
+  sort: 'accessibilitySort',
+  current: 'accessibilityCurrent',
+  dropEffect: 'accessibilityDropEffect',
+  grabbed: 'accessibilityGrabbed',
+  level: 'accessibilityLevel',
+  relevant: 'accessibilityRelevant',
+};
+
+export const supportedAccessibilityRoles = [
```

Comment:
maybe move this to `webToNativeAccessibilityRole.ts` file since it better belongs there

Reply:
```
const accessibilityRoleMap = {
  slider: 'adjustable',
  alert: 'alert',
  button: 'button',
  checkbox: 'checkbox',
  combobox: 'combobox',
  header: 'header',
  image: 'image',
  imagebutton: 'imagebutton',
  keyboardkey: 'keyboardkey',
  link: 'link',
  menu: 'menu',
  menubar: 'menubar',
  menuitem: 'menuitem',
  presentation: 'none',
  progressbar: 'progressbar',
  radio: 'radio',
  radiogroup: 'radiogroup',
  scrollbar:'scrollbar',
  search:'search',
  spinbutton:'spinbutton',
  region:'summary',
  switch:'switch',
  tab: 'tab',
  tablist: 'tablist',
  text: 'text',
  timer: 'timer',
  togglebutton: 'togglebutton',
  toolbar: 'toolbar',
  region: 'summary',
}
```